src/main/kotlin/org/koitharu/kotatsu/parsers/MangaParser.kt

```diff
@@ -62,14 +63,43 @@ public abstract class MangaParser @InternalParsersApi constructor(
 	@JvmField
 	protected val webClient: WebClient = OkHttpWebClient(context.httpClient, source)
 
+	/**
+	 * Search list of manga by specified searchQuery
+	 *
+	 * @param searchQuery searchQuery
+	 */
+	public suspend fun searchManga(searchQuery: MangaSearchQuery, validateQuery: Boolean = true): List<Manga> {
```

Comment:
Let's rename it to `getList`?
```suggestion
	public suspend fun getList(query: MangaSearchQuery): List<Manga> {
```

Reply:
Sure no problem